articles/tutorials/how-to-update-applications-client-secret.md

```diff
@@ -16,11 +16,11 @@ Once you have changed your application's `client_secret` you will need to update
 
 ## How to generate a new client_secret
```

Comment:
@hugoArregui With `PATCH secret endpoint` you mean this one? -> https://auth0.com/docs/api/management/v2#!/Clients/patch_clients_by_id

Also, I can't find this endpoint `POST /api/v2/clients/{id}/rotate-secret` in the [Auth0 Management API v2 page](https://auth0.com/docs/api/management/v2). Is the doc outdated? Thanks!